.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,6 @@
+contact_links:
+  - name: Usage Support
```

Comment:
i've always preferred to drive config and usage questions out of issues and into discussions, but our options havent really suggested that. this adds a link to the new issue step to at least present the option

potential downside: discussions are freeform, so we provide less guidance on providing relevant details. maybe https://github.blog/news-insights/product-news/github-discussions-just-got-better-with-category-forms/ is worth considering as a follow-up